app/views/companies/new.html.slim

```diff
@@ -0,0 +1,7 @@
+= provide(:button_text, 'Create new company')
+= provide(:cancel_btn_text, 'Cancel')
+= provide(:path, companies_path)
+
+h1.title New company
+.container
```

Comment:
Why you add here .container class? Do you want to use bootsttap grid here? If answer is yes, also should be .row, .col blocks